src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -406,12 +407,19 @@ const General = ({
             {
               dropdown: (
                 <Select className={styles.Field}>
-                  <Option value="20MHz">20MHz</Option>
-                  <Option value="40MHz">40MHz</Option>
-                  <Option value="80MHz">80MHz</Option>
+                  <Option value="is20MHz">is20MHz</Option>
+                  <Option value="is40MHz">is40MHz</Option>
+                  <Option value="is80MHz">is80MHz</Option>
```

Comment:
you have the `key` value right here just put the condition here 
`key === 'is2dot4GHz' ? null : <Option value="is80MHz">is80MHz</Option>`